buse_report_location.conf

```
@@ -0,0 +1,11 @@
+# Extracted from
+# https://github.com/matrix-org/mjolnir/blob/main/test/nginx.conf
+# Abuse reports should be sent to Mjolnir.
+# The r0 endpoint is deprecated but still used by many clients.
+# As of this writing, the v3 endpoint is the up-to-date version.
+# Alias the regexps, to ensure that they're not rewritten.
+set $room_id $2;
+set $event_id $3;
+# main-unit will be replace via charm by the main unit address
+# E.g.: synapse-0.model.endpoints
+proxy_pass http://synapse-0.synapse-endpoints:9999/api/1/report/$room_id/$event_id;
```

```suggestion
proxy_pass http://synapse-0.synapse-endpoints:9999/api/1/report/$room_id/$event_id;

```
final newline is missing